### DISCUSSION TITLE
changing fluid properties wont change the rel. perm. curves

### DISCUSSION THREAD

**Discussion by phscald** [PMEAL/OpenPNM#2854] [Q&A] [answered]
I am testing the demo code 'Predicting relative permeability' and have tried changing fluid properties to see how the curves behave.
However, the curves are not changing as they should. It looks like the answer is independent to fluid properties.

I am defining phase as 
`oil = op.phase.Phase(network=net)`
`oil['pore.viscosity'] = 0.01  # Pa.s`
`oil['pore.surface_tension'] = 5*10**(-3) #mN/m  #0.072`
`oil['pore.contact_angle'] = 30.0`
`oil.add_model_collection(op.models.collections.phase.standard_liquid)`
`oil.add_model_collection(op.models.collections.physics.basic)`
`oil.regenerate_models()`

water is the invading phase and is defined just as in the example.

The question has resemblance to the one in [https://github.com/PMEAL/OpenPNM/discussions/2842](url).

Do you guys have idea of why the change in phase do not change the flow behavior? I may be missing something.

I appreciate any help you might give.
Pedro

**Comment by jgostick** [comment, marked as answer]
This is the whole point of the "relative" part in relative permeability.  The "actual" permeability is a function of the fluid properties, but this is normalized out of the relative one.  The only thing that changes relative permeability is fluid configuration, so something like pore size distribution or variable wettability/contact angle distribution.  Or, you can use a different percolation algorithm that incorporates the effect of viscosity, which our basic percolation algorithms do not.

**Reply by phscald** [reply]
Thanks a lot. This theoretical error appeared because I was looking at flow rate curves before.
Have you implemented a dynamic algorithm, so I can get the flow rates over time? Or will I have to implement it?

**Reply by jgostick** [reply]
We have not got a dynamic IP, and it is quite challenging to implement. There have been a number of papers lately by groups in the oil and gas sphere that have reported on advances like GPU enabled ones, fully implicit ones, etc.
